articles/evaluation_queues.md

```diff
@@ -180,6 +180,15 @@ Clicking **Refresh Columns** will add these default columns.
 
 <img style="width: 80%;" src="/assets/images/leaderboard_columns.png">
 
+#### Leaderboard renderers
+
+* userid: Renders any Synapse user id to a Synapse user profile
+* date/epochdate: Renders epochtime in milliseconds to local time
```

Comment:
Note that in the web client, the user can select to show dates in UTC (in user settings) rather than local time.